README.md

```diff
@@ -67,7 +68,50 @@ For binary events, you will need to customize the Flink job with the appropriate
 Flink offers many options for customizing the behavior when writing files.
 Refer to [Steaming File Sink](https://ci.apache.org/projects/flink/flink-docs-release-1.10/dev/connectors/streamfile_sink.html)
 for details.
+## Configuring a seperate NFS mount to write files to NFS
+1. Add a PVC. ( Refer in FlinkCluster.yaml)
+```
+{{- if .Values.enableExtraVolume }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: flink-tools-nfs
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: nfs
```

Comment:
This will use the same Isilon used by Pravega LTS. The requirement is:
> Create a persistent volume that points to an arbitrary NFS mount and path. This must be different from any NFS mount used for Pravega LTS.
I believe you will need to create a new storage class that uses the NFS client provisioner that points to a different NFS server.
Also, don't worry about creating Helm templates for the PVC. Just create a sample yaml file. We'll let the user be responsible for that prior to running Helm.

Reply:
Storage Class is not required. We have already nfs storage class on our cluster. We can use the same.
chenna@chenna:~/trail/sdpt-533/flink-tools(feature-sdpt-533-write-files-to-nfs)$ k get sc
NAME                 PROVISIONER                            AGE
nfs                  cluster.local/nfs-client-provisioner   5d
standard (default)   kubernetes.io/vsphere-volume           5d5h